src/app/components/create-user-modal/createUserModal.js

```diff
@@ -6,15 +6,37 @@ import axios from "axios"
 // stocks is dict with companyName, symbol, id
 function CreateUserModal({stocks, onCreateUser}) {
 
-  const [selectedStock, setSelectedStock] = useState(null);
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [email, setEmail] = useState("");
+  const [selectedStock, setSelectedStock] = useState(stocks[0]);
```

Comment:
what is the reasoning behind making name/email/selectedStock to be state variables?